pkg/lightning/backend/kv/context.go

```diff
@@ -110,6 +110,11 @@ func (ctx *litExprContext) unsetUserVar(varName string) {
 	ctx.userVars.UnsetUserVar(varName)
 }
 
+// GetUserVar implements the `exprctx.BuildContext` interface.
+func (*litExprContext) IsReadonlyUserVar(string) bool {
```

Comment:
Why need to override this method? `*exprstatic.ExprContext` always return false.